src/main/scala/com/amazon/deequ/constraints/AnalysisBasedConstraint.scala

```diff
@@ -1,18 +1,18 @@
 /**
- * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"). You may not
- * use this file except in compliance with the License. A copy of the License
- * is located at
- *
- *     http://aws.amazon.com/apache2.0/
- *
- * or in the "license" file accompanying this file. This file is distributed on
- * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing
- * permissions and limitations under the License.
- *
- */
+  * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
How is this change related to the Spark/Scala update?

Reply:
Pulled this code over from PR #253 for scala 2.12 compatibility